src/components/DataSubmissions/MetadataUpload.tsx

```diff
@@ -5,7 +5,7 @@ import { LoadingButton } from "@mui/lab";
 import { isEqual } from "lodash";
 import { VariantType } from "notistack";
 import { Button, Stack, Typography, styled } from "@mui/material";
-import RadioInput from "./RadioInput";
+import Tooltip from "../Tooltip";
 import { CREATE_BATCH, CreateBatchResp, UPDATE_BATCH, UpdateBatchResp } from "../../graphql";
 import { useAuthContext } from "../Contexts/AuthContext";
 import DeleteDialog from "../../content/dataSubmissions/DeleteDialog";
```

Comment:
@Alejandro-Vega Can you just confirm that the DeleteDialog is still used for something in MetadataUpload? It seems like it isn't used anymore.

Reply:
Good catch, removed the dialog.